dockstore-client/src/main/java/io/dockstore/client/cli/Client.java

```diff
@@ -1177,6 +1194,256 @@ public static void updateContainerHelp() {
         out("");
     }
 
+    /**
+     * Finds the install location of the dockstore CLI
+     * @return
+         */
+    public static String getInstallLocation(){
+        String installLocation = null;
+
+        String executable = "dockstore";
+        String path = System.getenv("PATH");
+        String[] dirs = path.split(File.pathSeparator);
+
+        // Search for location of dockstore executable on path
+        for(String dir : dirs) {
+            // Check if a folder on the PATH includes dockstore
+            File file = new File(dir, executable);
+            if (file.isFile()) {
+                installLocation = dir + File.separator + executable;
+                break;
+            }
+        }
+
+        return installLocation;
+    }
+
+    /**
+     * Finds the version of the dockstore CLI for the given install location
+     * @param installLocation
+     * @return
+         */
+    public static String getCurrentVersion(String installLocation){
```

Comment:
Please clarify that this is to get the script version (as opposed to the version in the jar which can be obtained more simply). On the other hand, a thought about simplification occurs. Is there any situation in which the script and the jar have different versions? If there is none, then consider just getting the version from the jar which is simpler ex: https://github.com/SeqWare/seqware/blob/50d2ed3c449e88e43232e1af8a957ec8414441bd/seqware-webservice/src/main/java/net/sourceforge/seqware/webservice/resources/queries/EnvironmentResource.java#L35